src/barril/units/_tests/test_posc.py

```diff
@@ -431,3 +397,16 @@ def testJouleThomsonCoefficientUnits():
     assert joule_thomson_si_units_mega.GetValue("degC/bar") == 1e1
     assert joule_thomson_si_units_mega.GetValue("degF/Pa") == approx(9 / 5 * 1e6)
     assert joule_thomson_si_units_mega.GetValue("degF/bar") == approx(9 / 5 * 1e1)
+
+
+def testDensityDerivativePerTemperatureUnitConversion():
+    density_derivative_per_temperature = units.Scalar(
+        "density derivative in respect to temperature", 1, "kg/m3.degC"
+    )
+    assert density_derivative_per_temperature.value == 1.0
+    assert density_derivative_per_temperature.GetValue("kg/m3.K") == 1.0
+
+    density_derivative_per_temperature = units.Scalar(
+        "density derivative in respect to temperature", 1, "kg/m3.K"
+    )
+    assert density_derivative_per_temperature.GetValue("kg/m3.degC") == 1.0
```

Comment:
You probably know what you are doing but for me it is weird that 1 kg/m3 K is the same as 1 kg /m3 C.
I guess it is because degC and K have same scale but one has an offset of  ~273, right?

Reply:
Yes, the value remains the same after the conversion due to the scale. The derivative is a "variation". Since `1 delta K = 1 delta degC`, the conversion factor is equal to one.